firewood/src/merkle/node.rs

```diff
@@ -298,7 +313,6 @@ impl Node {
 struct Meta {
     root_hash: [u8; TRIE_HASH_LEN],
     attrs: NodeAttributes,
-    is_encoded_longer_than_hash_len: Option<bool>,
```

Comment:
I see the issue here. This is supposed to be the `NodeTypeId`, can we add that here.